manifestoes/extensions.rst

```diff
@@ -0,0 +1,1675 @@
+.. sectnum::
+.. contents::
+
+.. _PR: https://github.com/ghc-proposals/ghc-proposals/pull/628
+
+**GHC Language Configuration Renovation**
```

Comment:
This doesn't match the usual header? In particular the rendered version doesn't link back to the PR discussion at the top.

Reply:
It's not a usual proposal! It's a manifesto. :)

But I agree a direct link back to the PR would be helpful.